trin-core/src/utils.rs

```diff
@@ -42,19 +43,25 @@ mod test {
     }
 }
 
-pub fn get_data_dir() -> String {
-    let path = env::var(TRIN_DATA_ENV_VAR).unwrap_or_else(|_| get_default_data_dir());
+pub fn get_data_dir(local_enr: Enr) -> String {
+    let path = env::var(TRIN_DATA_ENV_VAR).unwrap_or_else(|_| get_default_data_dir(local_enr));
 
     fs::create_dir_all(&path).expect("Unable to create data directory folder");
     path
 }
 
-pub fn get_default_data_dir() -> String {
+pub fn get_default_data_dir(local_enr: Enr) -> String {
     // Windows: C:\Users\Username\AppData\Roaming\Trin\data
     // macOS: ~/Library/Application Support/Trin
     // Unix-like: $HOME/.local/share/trin
 
-    match ProjectDirs::from("", "", "Trin") {
+    // Append last 8 enr base64 encoded chars to application dir name
+    let mut application_string = "Trin_".to_owned();
+    let len = &local_enr.to_base64().len();
+    let suffix = &local_enr.to_base64()[len - 8..];
```

Comment:
If I were new to trin, and found a directory named like this, I would expect it to be the first few characters of the as-displayed node-id. I understand if that feels like a tangent to you right now, and if so you want to skip it. In that case, maybe you could add a "good first issue" with that suggestion? Seems like a nice bite-size chunk for a newbie who wants to get started.